### PR TITLE
updated model path for dolly-v2-12

### PR DESCRIPTION
**Pull request by choudharykartik**
endpoint link: https://ml.azure.com/endpoints/realtime/dolly-12b-endpoint-gpu-test/detail?wsid=/subscriptions/b17253fa-f327-42d6-9686-f3e553e24763/resourcegroups/win19/workspaces/11120925024Nov&tid=72f988bf-86f1-41af-91ab-2d7cd011db47
response 
image

batch deployment link: https://ml.azure.com/experiments/id/2ad8bd8f-fbda-434e-8f3a-e81cc5780829/runs/batchjob-784d9f0e-fd0e-4368-817c-67dfb7ab2c4e?wsid=/subscriptions/b17253fa-f327-42d6-9686-f3e553e24763/resourceGroups/win19/providers/Microsoft.MachineLearningServices/workspaces/11120925024Nov&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#

text-classification run link: https://ml.azure.com/experiments/id/521bd67c-d71b-47bf-9b2c-f628973f3d4b/runs/good_curtain_7625tf759w?wsid=%2Fsubscriptions%2Fb17253fa-f327-42d6-9686-f3e553e24763%2Fresourcegroups%2Fwin19%2Fworkspaces%2F11120925024Nov&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#

token classfication run link: https://ml.azure.com/runs/brave_gold_xzphsgvvxz?wsid=/subscriptions/b17253fa-f327-42d6-9686-f3e553e24763/resourcegroups/win19/workspaces/11120925024Nov&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#

eval run link: https://ml.azure.com/experiments/id/dd332e12-ceee-4be5-9394-b9e3785916d4/runs/tough_octopus_k414yvmx1n?wsid=%2Fsubscriptions%2F72c03bf3-4e69-41af-9532-dfcdc3eefef4%2FresourceGroups%2Fshared-finetuning-rg%2Fproviders%2FMicrosoft.MachineLearningServices%2Fworkspaces%2Fv-kartik2&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#

save model to blob link: https://ml.azure.com/runs/coral_sprout_3d16w4lzf1?wsid=/subscriptions/b17253fa-f327-42d6-9686-f3e553e24763/resourcegroups/win19/workspaces/11120925024Nov&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#

### CODE REVIEW

File: assets/models/system/databricks-dolly-v2-12b/spec.yaml [hunk 1]
```diff
@@ -4,10 +4,12 @@ path: ./
 properties:
   SHA: 19308160448536e378e3db21a73a751579ee7fdd
   datasets: databricks/databricks-dolly-15k
+  finetuning-tasks: text-classification, token-classification
```
Comment (novaturient95) [inline assets/models/system/databricks-dolly-v2-12b/spec.yaml:7]: 🚀

**Comment by novaturient95** [comment]
Lets update description with finetuning samples and new recorded inference samples

File: assets/models/system/databricks-dolly-v2-12b/spec.yaml (hunk 1 above)
Comment (novaturient95) [reply assets/models/system/databricks-dolly-v2-12b/spec.yaml:7]: text-gen has issue. For which there is a bug on FT team: https://msdata.visualstudio.com/Vienna/_workitems/edit/2835559

**Comment by choudharykartik** [comment]
> Lets update description with finetuning samples and new recorded inference samples

updated, Thanks

**Pull request merged**